website/blog/2023-09-17-databend-weekly-111.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "This Week in Databend #111"
+date: 2023-09-17
+slug: 2023-09-17-databend-weekly
+cover_url: 'weekly/weekly-111.jpg'
+image: 'weekly/weekly-111.jpg'
+tags: [weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: Chasen-Zhang
+  - name: ct20000901
+  - name: dantengsky
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: everpcpc
+  - name: gitccl
+  - name: JackTan25
+  - name: nagarajatantry
+  - name: nange
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: ZhiHanZ
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Understanding SHARE ENDPOINT
+
+SHARE ENDPOINT is an important concept in Databend data sharing, used to specify the endpoint and tenant name of a data sharing instance. The accessors and users of the data need to define SHARE POINT to help the Databend instance access the shared data.
```

Comment:
"SHARE ENDPOINT" is a concept in Databend data sharing, as it defines the endpoint and tenant name of a data sharing instance. Consumers of the shared data must define a "SHARE ENDPOINT" in order to access the shared data.